python/cudf/cudf/_lib/column.pyi

```diff
@@ -0,0 +1,121 @@
+from typing import Tuple, Union, TypeVar, Optional
+
+from cudf._typing import DtypeObj, Dtype, ScalarObj
+from cudf.core.buffer import Buffer
+from cudf.core.column import ColumnBase
+
+
+T = TypeVar("T")
+
+class Column:
+    _data: Union[Buffer, None]
+    _mask: Buffer
+    _base_data: Buffer
+    _base_mask: Buffer
```

Comment:
Should these be `Union[Buffer, None]`?

Reply:
Connected offline, should create a type `BufferOrNone` and use that

Reply:
Went with your suggestion of using `Optional[Buffer]` instead, as that is MyPy's recommendation.